packages/ferric/src/template.ts

```diff
@@ -0,0 +1,58 @@
+export function cargoTomlTemplate(projectName: string) {
+	return `
+[package]
+name = ${projectName}
+version = "1.0.0"
+edition = "2021"
+license = "MIT"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies.napi]
+version = "3.1"
+default-features = false
+# see https://nodejs.org/api/n-api.html#node-api-version-matrix
+features = ["napi4"]
+
+[dependencies.napi-derive]
+version = "3.1"
+features = ["type-def"]
+
+[build-dependencies]
+napi-build = "2"
+
+[profile.release]
+lto = true
+codegen-units = 1
+strip = "symbols"
+opt-level = "z"
+panic = "abort"
```

Comment:
Although this does make a small release build, some of these optimisations have big consequences (e.g. `panic = "abort"`) so I probably wouldn't make them defaults.